lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -141,6 +140,21 @@ class window.Turbolinks
 
     return
 
+  getNodesMatchingRefreshKeys = (keys) ->
+    matchingNodes = []
+    for key in keys
+      for node in document.querySelectorAll("[refresh=#{key}]")
+        matchingNodes.push(node)
+
+    return matchingNodes
+
+  getNodesWithRefreshAlways = ->
+    matchingNodes = []
+    for node in document.querySelectorAll("[refresh-always]")
+      matchingNodes.push(node)
+
+    return matchingNodes
```

Comment:
https://developer.mozilla.org/en/docs/Web/API/NodeList I don't think we can use map lol